src/components/CompareTable/CompareCompetencesTableRow.tsx

```diff
@@ -0,0 +1,35 @@
+import { components } from '@/api/schema';
+import { capitalize, getLocalizedText } from '@/utils';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export interface CompareCompetencesTableRowData {
+  uri: string;
+  nimi: components['schemas']['LokalisoituTeksti'];
+  kuvaus: components['schemas']['LokalisoituTeksti'];
+  profiili?: boolean;
+}
+
+interface CompareCompetencesTableRowProps {
+  row: CompareCompetencesTableRowData;
+  className?: string;
+}
+
+export const CompareCompetencesTableRow = ({ row, className }: CompareCompetencesTableRowProps) => {
+  const { t } = useTranslation();
+  const title = React.useMemo(() => getLocalizedText(row.kuvaus), [row.kuvaus]);
+
+  return (
+    <tr className={className}>
+      <td className="w-full pl-5 pr-7 py-3 text-heading-4 hyphens-auto" title={title}>
+        {capitalize(getLocalizedText(row.nimi))}
+      </td>
+      <td className="justify-items-center pr-7">
+        <div role="img" aria-label={t('found')} className="h-5 w-5 rounded-full bg-secondary-4" />
+      </td>
+      <td className="justify-items-center pr-5">
+        {row.profiili && <div role="img" aria-label={t('found')} className="h-5 w-5 rounded-full bg-[#333]" />}
```

Comment:
Just a small thing, but the "dot" could be component with the color changing; preventing duplication and easier maintaining if there are changes later on.

Using `className` to preserve the Tailwind plugin's ability to suggest tokens.

```tsx
const FoundSkillDot = ({ className }: { className: string }) => {
  const { t } = useTranslation();
  return <div role="img" aria-label={t('found')} className={`h-5 w-5 rounded-full ${className}`} />;
};

// ...
        <FoundSkillDot color="bg-secondary-4" />
        <FoundSkillDot color="bg-[#333]" />
```

There is a SonarLint warning related to the used `img='role'`, not solved in this example